### ISSUE TITLE
Fine-tune descriptor mismatch warning should ignore seed

### ISSUE THREAD

**Issue by OutisLi** [deepmodeling/deepmd-kit#6038]
## Summary

`warn_configuration_mismatch_during_finetune` reports `descriptor.seed` as a configuration mismatch. `seed` only selects the RNG used when a module is first constructed. It is not a loaded parameter, and a difference does not change which pretrained weights are reused.

## What happens

`_IGNORED_DESCRIPTOR_KEYS` in `deepmd/utils/finetune.py` is `{"trainable"}`. A fine-tune whose input sets `descriptor.seed` to `null` while the checkpoint was trained with `seed: 42` therefore logs:

```text
DEEPMD WARNING Descriptor configuration mismatch detected between input.json and pretrained model (branch 'Default'). Only descriptor parameters that are compatible with the pretrained model can be reused; incompatible parameters may be reinitialized, skipped, or rejected by backend-specific loading:
  seed: input=None, pretrained=42
```

The weight copy in `collect_single_finetune_params` (`deepmd/pt/train/training.py`) does not consult this diff. Matching tensors are still taken from the pretrained state dict. The warning text implies that the descriptor may be reinitialized or rejected, which does not happen for `seed`.

`descriptor.seed: null` is the documented way to draw a fresh seed for any parameter that is actually created during the run. That choice should not be reported as an incompatible descriptor change.

## Expected behavior

Add `seed` to `_IGNORED_DESCRIPTOR_KEYS`, next to `trainable`, so initialization RNG settings are not treated as architecture mismatches.

**Comment by OutisLi** [comment]
`descriptor.use_amp` belongs in the same ignore set.

It is an execution policy, not part of the learned function. The DPA4C serializer already leaves it out of the checkpoint for that reason (`deepmd/dpmodel/descriptor/dpa4c.py`, `serialize`): training reads it from the input, and inference reads `DP_AMP_INFER`. The pt / pt_expt modules store it as a Python bool (`self.use_amp = bool(use_amp)`), so `load_state_dict` does not restore it and the input value is what actually enables autocast.

Fine-tuning an AMP checkpoint with `descriptor.use_amp: false` therefore still loads the pretrained weights, but logs:

```text
DEEPMD WARNING Descriptor configuration mismatch detected between input.json and pretrained model (branch 'Default'). Only descriptor parameters that are compatible with the pretrained model can be reused; incompatible parameters may be reinitialized, skipped, or rejected by backend-specific loading:
  use_amp: input=False, pretrained=True
```

That warning is wrong in the same way as `seed`: nothing is reinitialized or rejected. Please ignore `use_amp` in `_IGNORED_DESCRIPTOR_KEYS` as well.
